.rubocop.yml

```diff
@@ -148,10 +148,18 @@ Lint/DeprecatedOpenSSLConstant:
   Description: "Don't use algorithm constants for `OpenSSL::Cipher` and `OpenSSL::Digest`."
   Enabled: true
 
+Lint/DuplicateBranch:
+  Description: Checks that there are no repeated bodies within `if/unless`, `case-when` and `rescue` constructs.
+  Enabled: false
```

Comment:
This is enabled automatically. I've disabled it because it's a heavy change and requires a separate PR if we want to enable it. It basically checks for if branches that do the same thing, but it can't be fixed automatically and requires code restructure